app/components/show/agreement/details_component.html.erb

```diff
@@ -36,13 +36,6 @@
         </td>
       </tr>
 
-      <tr>
-        <th class="col-3" scope="row">Released to</th>
-        <td>
-            <%= released_to %>
```

Comment:
Can you remove the `released_to` method from Agreement::DetailsComponent now that it is not used?

Reply:
yes!

Reply:
done.  Ready for re-review, @jcoyne